src/dynamic.cpp

```diff
@@ -28,10 +28,10 @@ extern "C" {
   ::dynamicgraph::Entity*						
   EntityMaker_Dynamic(const std::string& objname)			
```

Comment:
This should be changed to EntityMaker_DynamicPinocchio as the new entity name should match Class Type